content/CORE/CORETutorials/Services/ConfigureDynamicDNS.md

```diff
@@ -16,4 +19,4 @@ Log in to the TrueNAS web interface and go to **Services > Dynamic DNS**.
 The required values for these fields are provided by your DDNS solution.
 Start the DDNS service after choosing your **Provider** options and saving the settings.
 
-For more information on the configuration settings see  [**Dynamic DNS** screen]({{< relref "/CORE/UIReference/Services/DynamicDNSScreen.md" >}}).
\ No newline at end of file
+{{< taglist tag="coreddns" limit="10" title="Related Articles" >}}
```

Comment:
it is my understanding that using the title="Related xyz Articles" tag is only necessary if you are using the tag a second time to cover other related articles where you want to say essentially what the tag topic is. For example, the article is on DDNS but you want to show Related IP Articles and you use the coreip tag.

Reply:
Removed taglist title parameter, updated image to 13.0 directory, changed description to be more standardized.

Reply:
These issues have been addressed in the corrections.